packages/pn-personafisica-webapp/src/component/Notifications/NotificationPayment.tsx

```diff
@@ -30,7 +30,10 @@ import { useTranslation } from 'react-i18next';
 import { useAppDispatch, useAppSelector } from '../../redux/hooks';
 import { getNotificationPaymentInfo, getPaymentAttachment } from '../../redux/notification/actions';
 import { RootState } from '../../redux/store';
-import { PAGOPA_HELP_EMAIL, PAYMENT_DISCLAIMER_URL } from '../../utils/constants';
+import { PAGOPA_HELP_EMAIL,
+  // PN-2029
+  // PAYMENT_DISCLAIMER_URL
```

Comment:
Controllare se questa variabile viene usata da altre parti. Se no, eliminare (commentare va bene) i riferimenti in constants e nel buildspec di persona fisica (REACT_APP_PAYMENT_DISCLAIMER_URL)